DataDog/datadog-agent
# [process-agent]Add chunking for container processes payloads sent

### What does this PR do?

Add chunking for container processes payloads sent. This is to avoid payloads that are too large for ingestion.

### Motivation

What inspired you to submit this pull request?

### Describe your test plan
- Enable debug logging in `datadog.yaml`
```
logs_enabled: true
log_level: 'debug'
```
- Run the agent in an environment with containers.
  - Example images:
    - https://hub.docker.com/_/busybox 
    - https://hub.docker.com/_/nginx
- Check Live Processes and Containers has their respective data
- Check logs for how many messages are created for containers
   ```
   (pkg/process/checks/process.go:263 in packProcCtrMessages) | Created 1 container process messages
   ```
- Add/Update this config to change the chunking
```
process_config:
  max_ctr_procs_per_message: 1 # this will force all containers to be sent on their own message.
```
